pkg/worker/redis.go

```diff
@@ -206,8 +209,10 @@ func (w *RedisWorker) processJob(ctx context.Context, job *Job) {
 	logger.Info().Msg("Dequeued job from Redis")
 	ctx = contextLogger(ctx, job)
 	if h, ok := w.handlers[job.Type]; ok {
+		cCtx, cFunc := context.WithTimeout(ctx, config.Worker.Timeout)
+		defer cFunc()
 		metrics.ObserveBackgroundJobDuration(job.Type.String(), func() {
-			h(ctx, job)
+			h(cCtx, job)
 		})
```

Comment:
These three lines are actually the change + configuration option.

All the rest are just tests :-)

Reply:
Shall we add this to memory implementation as well?
You know, just for consistency...

Reply:
Memory is only used for development, I don’t see any benefit but why not.